.github/workflows/go_mod_tidy.yml

```
@@ -1,17 +1,25 @@
 name: "Run Go Mod Tidy And Generate Licenses"
 on:
+  pull_request:
+    types:
+      - labeled
   workflow_dispatch:
     inputs:
       pr_number:
-        description: 'PR number'     
+        description: 'PR number'
         required: true
         type: number
+permissions:
+  contents: write
 jobs:
+  if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependencies-go')) }}
```

We can do this
```suggestion
  if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.actor == 'dependabot[bot]') }}
```

and then check `${{ contains(github.event.pull_request.labels.*.name, 'dependencies-go')}}` only for doing `inv -e generate-licenses`